src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -2596,7 +2596,11 @@ private bool getTailCallHelpers(ref CORINFO_RESOLVED_TOKEN callToken, CORINFO_SI
         {
             // Slow tailcalls are not supported yet
             // https://github.com/dotnet/runtime/issues/35423
+#if READYTORUN
```

Comment:
Interesting - if this is reachable in READYTORUN, throwing a `NotImplemented` here tears down the process because only some specific exceptions are catchable/recoverable (type system exceptions or RequiresRuntimeJitException).

Reply:
cc @trylek Do we see crossgen2 tests failing on this? We may want to change this to `RequiresRuntimeJitException`.